charts/cht-chart-4x/Chart.yaml

```diff
@@ -2,25 +2,10 @@ apiVersion: v2
 name: cht-chart-4x
 description: A Helm chart for Community Health Toolkit (CHT) v4
 home: https://github.com/medic/cht-core
+icon: https://avatars.githubusercontent.com/u/474424?s=200&v=4
 
-# A chart can be either an 'application' or a 'library' chart.
-#
-# Application charts are a collection of templates that can be packaged into versioned archives
-# to be deployed.
-#
-# Library charts provide useful utilities or functions for the chart developer. They're included as
-# a dependency of application charts to inject those utilities and functions into the rendering
-# pipeline. Library charts do not define any templates and therefore cannot be deployed.
 type: application
 
-# This is the chart version. This version number should be incremented each time you make changes
-# to the chart and its templates, including the app version.
-# Versions are expected to follow Semantic Versioning (https://semver.org/)
 version: 1.0.0
 
-
-# This is the version number of the application being deployed. This version number should be
-# incremented each time you make changes to the application. Versions are not expected to
-# follow Semantic Versioning. They should reflect the version the application is using.
-# It is recommended to use it with quotes.
-appVersion: "1.16.0"
+appVersion: ""
```

Comment:
was this intentionally removed?

Reply:
It's intentional, I don't think it adds any value could be confusing, for reference appVersion 1.16.0 is an artefact created from `helm create x` that references the nginx version of the example pod